src/presentation/http/router/noteSettings.test.ts

```diff
@@ -10,11 +10,25 @@ describe('NoteSettings API', () => {
       const existingNotePublicId = 'Pq1T9vc23Q';
 
       const expectedNoteSettings = {
-        'customHostname': 'codex.so',
-        'isPublic': true,
-        'id': 2,
-        'noteId': 2,
-        'invitationHash': 'E2zRXv3cp-',
+        'noteSettings':  {
+          'customHostname': 'codex.so',
+          'id': 2,
+          'invitationHash': 'E2zRXv3cp-',
+          'isPublic': true,
+          'noteId': 2,
+        },
+        'team':  [
```

Comment:
Why array? there can be more than 1 team per note?

Reply:
For now note can have only one team